packages/react-email/src/cli/utils/start-server-command.ts

```diff
@@ -11,8 +15,57 @@ function execAsync(command: string) {
   });
 }
 
-export const startDevServer = (packageManager: string, port: string) => {
-  execAsync(`${packageManager} run dev -- -p ${port}`);
+export const startDevServer = async (_packageManager: string, port: string) => {
+  const isRunningBuilt = __filename.endsWith('index.js');
+  const app = next({
+    dev: true,
+    hostname: 'localhost',
+    port: parseInt(port),
+    customServer: true,
+    dir: isRunningBuilt ? path.resolve(__dirname, '..') : path.resolve(__dirname, '../../..'),
+  });
+
+  await app.prepare();
+
+  const nextHandleRequest = app.getRequestHandler();
+
+  http
+    .createServer((req, res) => {
+      if (!req.url) {
+        res.end(404);
+        return;
+      }
+
+      const parsedUrl = url.parse(req.url, true);
+
+      // Never cache anything to avoid
+      res.setHeader(
+        'Cache-Control',
+        'no-cache, max-age=0, must-revalidate, no-store',
+      );
+      res.setHeader('Pragma', 'no-cache');
+      res.setHeader('Expires', '-1');
+
+      try {
+        void nextHandleRequest(req, res, parsedUrl);
+      } catch (e) {
+        console.error('caught error', e);
+
+        res.writeHead(500);
+        res.end();
+      }
+    })
+    .listen(port, () => {
+      console.log(`running preview at localhost:${port}`);
```

Comment:
Add the `http://` so it's clickable in the CLI.

```suggestion
      console.log(`running preview at http://localhost:${port}`);
```